### PR TITLE
DEVX-1556: Adapts Find-Distinct tutorial for ksql 0.7 changes

### PR DESCRIPTION
**Pull request by bbejeck**
Verified this by running through all the tutorial steps and run `make` locally.

### CODE REVIEW

File: _data/harnesses/finding-distinct/ksql.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ dev:
         - action: docker_ksql_cli_session
           container: ksql-cli
           docker_bootup_file: tutorial-steps/dev/start-cli.sh
+          column_width: 73
```
Comment (bbejeck) [inline _data/harnesses/finding-distinct/ksql.yml]: This was needed to make the rendering as best as possible

File: _includes/tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/create-inputs/output-0.log [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Message        
-----------------
+
```
Comment (bbejeck) [inline _includes/tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/create-inputs/output-0.log]: Not sure these intermediate output files matter.  I guess we'll address this in a follow-up PR where remove the output files from source control.  This applies to several output files in this PR.

File: _includes/tutorials/finding-distinct/ksql/markup/dev/print-topic.adoc [hunk 3]
```diff
@@ -7,5 +7,5 @@ Finally, let's see what's available on the underlying Kafka topic for the table.
 The output should look similar to:
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/print-topic/output-0.log %}</code></pre>
-+++++
\ No newline at end of file
+<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/expected-print-topic.log %}</code></pre>
```
Comment (bbejeck) [inline _includes/tutorials/finding-distinct/ksql/markup/dev/print-topic.adoc]: updated this to not look at output directory but expected outputs, as this is how it should be for the tutorial IMHO.

File: _includes/tutorials/finding-distinct/ksql/markup/dev/transient-query.adoc [hunk 4]
```diff
@@ -11,5 +11,5 @@ To verify everything is working as expected, run the following query:
 The output should look similar to:
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/expected-transient-query.log%}</code></pre>
```
Comment (bbejeck) [inline _includes/tutorials/finding-distinct/ksql/markup/dev/transient-query.adoc]: same as above

File: _includes/tutorials/finding-distinct/ksql/markup/dev/transient-window.adoc [hunk 5]
```diff
@@ -7,5 +7,5 @@ Let's experiment with these events. We need to create a query capable of display
 This query should produce the following output:
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/transient-window/output-0.log %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/expected-transient-window.log%}</code></pre>
```
Comment (bbejeck) [inline _includes/tutorials/finding-distinct/ksql/markup/dev/transient-window.adoc]: same as above

File: _includes/tutorials/finding-distinct/ksql/markup/dev/print-topic.adoc (hunk 3 above)
Comment (rspurgeon) [reply _includes/tutorials/finding-distinct/ksql/markup/dev/print-topic.adoc]: Agree, we've done this in some others but I'm not sure if we've covered it across the board

File: _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml [hunk 6]
```diff
@@ -64,16 +66,16 @@ services:
...
   ksql-cli:
-    image: confluentinc/cp-ksql-cli:5.3.2
+    image: confluentinc/ksqldb-server:0.7.1
     container_name: ksql-cli
     depends_on:
       - broker
       - ksql-server
     entrypoint: /bin/sh
+    environment:
+      KSQL_CONFIG_DIR: "/etc/ksql"
```
Comment (rspurgeon) [inline _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml]: We might want to update the rest of the service versions in this file to be what the others are, which is 5.4.0 I believe

Comment (bbejeck) [reply _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml]: ack

**Comment by bbejeck** [comment]
I'll wait to confirm the test passed for this PR, then merge

**Comment by colinhicks** [comment]
Most recent test run passed.

**Pull request merged**